.gitignore

```diff
@@ -11,6 +11,7 @@ ui_*.h
 *.pro.user*
 qrc_*
 docs/html
+.directory
```

Comment:
I will do that. Thank you for the information. I wasn't aware I could do
that. :)
On Jan 10, 2015 1:48 PM, "Thorbjørn Lindeijer" <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/bjorn/tiled/pull/859#discussion-diff-22762199>:
>
> > @@ -11,6 +11,7 @@ ui_*.h
> >  *.pro.user*
> >  qrc_*
> >  docs/html
> > +.directory
>
> Ok, then please set this as a global ignore, for example by putting it in
> ~/.gitignore and doing:
>
> git config --global core.excludesfile ~/.gitignore
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bjorn/tiled/pull/859/files#r22762199>.
>